src/Controls/src/Core/Stepper/Stepper.cs

```diff
@@ -45,7 +45,7 @@ public partial class Stepper : View, IElementConfiguration<Stepper>, IStepper
 		//'-log10(increment) + 4' as rounding digits gives us 4 significant decimal digits after the most significant one.
 		//If your increment uses more than 4 significant digits, you're holding it wrong.
 		/// <summary>Bindable property for <see cref="Increment"/>.</summary>
-		public static readonly BindableProperty IncrementProperty = BindableProperty.Create(nameof(Increment), typeof(double), typeof(Stepper), 1.0,
+			public static readonly BindableProperty IncrementProperty = BindableProperty.Create(nameof(Increment), typeof(double), typeof(Stepper), 1.0,
```

Comment:
Maybe just revert this when doing the test updates?

Reply:
Reverted this change